src/pybind/mgr/dashboard/controllers/multi_cluster.py

```diff
@@ -266,6 +268,15 @@ def is_token_expired(self, jwt_token):
         current_time = time.time()
         return expiration_time < current_time
 
+    def get_time_left(self, jwt_token):
+        split_message = jwt_token.split(".")
+        base64_message = split_message[1]
+        decoded_token = json.loads(base64.urlsafe_b64decode(base64_message + "===="))
```

Comment:
Can you give example of JWT token you are receiving in comments.

Reply:
@cloudbehl , something like this `eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE3MDk4Mjg4NjksImlhdCI6MTcwOTgwMDA2OSwiaXNzIjoiY2VwaC1kYXNoYm9hcmQiLCJqdGkiOiI5YTAwNjM1Ni0zZmQzLTQzOGUtYjY3OS1mMzEwZjlhNTBiZGUiLCJ1c2VybmFtZSI6ImFkbWluIn0.D6tj8D4MAfvF0Rky8qzmSbZF5YvqrH1UuQZXypmJP2Q`